src/ontology/make-bridge-ontologies-from-xrefs.pl

```diff
@@ -74,6 +83,9 @@
                 }
                 elsif ($t eq 'gd') {
                     print $fh "intersection_of: $id ! $n\n";
+                    if (exists($lsxrefs{$x})) {
+                        $rel = "BFO:0000066";
```

Comment:
Just for consistency, can we declare occurs_in same way as we do part of at the end of the file (https://github.com/obophenotype/uberon/blob/master/src/scripts/make-bridge-ontologies-from-xrefs.pl#L113)

Reply:
Sure.